galaxy/templates/deployment-workflow.yaml

```diff
@@ -39,6 +39,11 @@ spec:
           volumeMounts:
             - name: galaxy-data
               mountPath: {{ .Values.persistence.mountPath }}
+        {{ if .Values.rabbitmq.enabled }}
+        - name: {{ .Chart.Name }}-init-rabbitmq
+          image: alpine:3.7
+          command: ['sh', '-c', 'until nc -z -w3 {{ template "galaxy-rabbitmq.fullname" $ }} 5672; do echo "waiting for galaxy-rabbitmq service"; sleep 1; done;']
```

Comment:
@pcm32 Does this mean that Galaxy will not startup if rabbitmq is not up?

Reply:
If you set the ampq connection to rabbit and rabbit is not up, then Galaxy fails if rabbit is not there where expected, so we need to wait for it.